auditor/src/domain/score.rs

```diff
@@ -13,6 +13,24 @@ use serde::{Deserialize, Serialize};
 use sqlx::postgres::PgHasArrayType;
 use std::cmp::Ordering;
 
+/// [`Score`]s are attached to a [`Component`](`crate::domain::Component`)
+/// and are used to relate different components of the same kind to each other sin some
+/// kind of performance characteristic.
```

Comment:
Should "sin" be "such as"?

Reply:
It should be "in", i.e. "... to each other in some kind of ..."
Fixed.